mycli/packages/expanded.py

```diff
@@ -8,6 +9,16 @@ def get_separator(num, header_len, data_len):
     sep = u"***************************[ %d. row ]***************************\n" % (num + 1)
     return sep
 
+def format_field(value):
+    # Returns the field as a text type, otherwise will hexify the string
+    try:
+        if isinstance(value, bytes):
+            return _text_type(value, "ascii")
+        else:
+            return _text_type(value)
+    except UnicodeDecodeError:
+        return _text_type('0x' + (codecs.getencoder('hex_codec')(value)[0]).decode('ascii'))
```

Comment:
yeah binascii works just as well and is quite a bit prettier. I'll swap it out and run some tests to make sure it actually works the same way!